recipes/grep/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "grep" %}
+{% set version = "3.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://mirrors.kernel.org/gnu/{{ name }}/{{ name }}-{{ version }}.tar.xz
+  sha256: abcd11409ee23d4caf35feb422e53bbac867014cfeed313bb5f488aca170b599
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - make
+  host:
+    - pcre2
+
+test:
+  commands:
+    - ${CONDA_PREFIX}/bin/grep --version
+    - echo hello world | ${CONDA_PREFIX}/bin/grep hello
```

```suggestion
    - test -x ${PREFIX}/bin/grep
    - if [ $(which grep) != ${PREFIX}/bin/grep ]; then echo 1; fi
    - grep --version
    - echo hello world | grep hello
```

one would use `${PREFIX}` in recipes, and `${CONDA_PREFIX}` during other usage.

---

thanks, makes sense!